wallet/src/ui/app/index.tsx

```diff
@@ -57,6 +61,15 @@ const App = () => {
             <Route path="/*" element={<HomePage />}>
                 <Route path="tokens" element={<TokensPage />} />
                 <Route path="nfts" element={<NftsPage />} />
+                <Route path="apps" element={<AppsPage />} />
+                <Route
```

Comment:
I think it'd be more ideal to have `apps/*` direct to the apps page, then let apps have its own child route definitions.